lineapy/utils/config.py

```diff
@@ -5,6 +5,9 @@
 
 LOG_FILE_NAME = "linea.log"
 
+CUSTOM_ANNOTATIONS_FOLDER_NAME = "custom-annotations"
+
+CUSTOM_ANNOTATIONS_EXTENSION_NAME = ".annotations.yaml"
```

Comment:
not sure if i mentioned this before but all these string constants might be better organized in the constants file. That way we dont have to look around for these.

Reply:
I put these constants here because `FOLDER_NAME` and `LOG_FILE_NAME` are declared here. I can move
my constants to the constants file.